jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/complex/DatePickersTests.java

```diff
@@ -449,15 +446,18 @@ public void testInternationalizationDatePicker() {
         }
         jdiAssert(shownChineseDaysOfWeek, containsInAnyOrder(CHINESE_DAYS_OF_WEEK.toArray()),
                 "For Chinese picker: shown and expected days of week are not the same");
+
+        chineseDatePicker.root().show();
         chineseDatePicker.changeMonth();
-        waitCondition(() -> chineseDatePicker.getChangeYearButton().isVisible());
+        waitCondition(() -> chineseDatePicker.getChangeYearButton().isClickable());
         chineseDatePicker.has().visibleChangeYearButton();
         jdiAssert(chineseDatePicker.getAllMonths(), is(CHINESE_MONTHS),
                 "For Chinese picker: shown and expected days of week are not the same");
```

Comment:
Just to think about :) in ideal world we have all asserts in ***Assert.java
Maybe it's feasible here :)

Reply:
Do you mean, that I should avoid using jdiAssert and move this assert to DatePickerAssert class?
chineseDatePicker.has().properShownMonth(expectedMonth, actualMonth);
properShownMonth(List<String> expectedMonth, List<String> actualMonth) {...}